packages/blade/src/components/AutoComplete/_decisions/decisions.md

```diff
@@ -0,0 +1,240 @@
+| accessibilityLabel | aria-label of input when label is not defined                                                                        | string                                                          |                   |
+| rows               | height restrictions of input ([Checkout "Inactive - Active States" Usage](#with-different-inactive---active-states)) | '1' \| '3' \| 'expandable'                                      | '3'               |
+| showTags           | Boolean to show or hide tags inside AutoComplete ([Checkout "With Tags Outside" Usage](#with-tags-outside))          | boolean                                                         | true              |
+| value              | Controlled state of the value inside input                                                                           | string                                                          |                   |
+| onChange           | Callback when input value changes                                                                                    | (inputValue: string) => void                                    |                   |
+| selected           | Controlled state of the selected items ([Checkout "Controlled AutoComplete" Usage](#controlled-autocomplete))        | string[]                                                        |                   |
+| onSelectChange     | Callback when selected items change                                                                                  | ({ name, values }: { name: string; values: string[] } ) => void |                   |
+
+## Usage
+
+AutoComplete will be built as a new trigger for existing Dropdown component.
+
+Depending on where and how AutoComplete is used, the API can differ. Checkout examples below on how code will look like for a particular usage.
+
+- [Basic Usage](#basic-usage)
+- [Controlled AutoComplete](#controlled-autocomplete)
+- [In BottomSheet](#in-bottomsheet)
+- [With Tags Outside](#with-tags-outside)
+- [With Different Inactive - Active States](#with-different-inactive---active-statess)
+
+### Basic Usage
+
+```jsx
+<Dropdown selectionType="multiple">
+  <AutoComplete label="Select Cities" />
+  <DropdownOverlay>
+    <ActionList>
+      <ActionListItem title="Mumbai" value="mumbai" />
+      <ActionListItem title="Pune" value="pune" />
+      <ActionListItem title="Hyderabad" value="hyderabad" />
+    </ActionList>
+  </DropdownOverlay>
+</Dropdown>
+```
+
+### Controlled AutoComplete
+
+There are 3 controlled states that are relevant to AutoComplete
+
+1. Controlled state of filtering items
+2. Controlled state of the Input where user types
+3. Controlled state of selected items
+
+```jsx
+<AutoComplete
+  // For controlling the input value
+  value={}
```

Comment:
what are the cases for value and selected as 2 diff things? because what is selected will be displayed in the input right? 🤔

Reply:
In this value is `"O"` and selected is `['apple', 'bob-gordon-american-elderberry']`
![image](https://github.com/razorpay/blade/assets/30949385/fffa11fa-1ecf-48e7-a07c-1f277b73e5aa)


We need 2 states because in 1 state, we can't differentiate between what is being written inside input vs what is selected.